### DISCUSSION TITLE
Deletion of account

### DISCUSSION THREAD

**Discussion by flefevre** [open-webui/open-webui#5317] [Q&A]
Dear all,
i would like to know what is the procedure to delete account in openwebui.
I have succeed in configuring the SSO but previous account are configured without sso.
Thanks for the tips.

**Comment by flefevre** [comment]
Any idea how to delete an user account from the admin panel?

**Reply by silentoplayz** [reply]
![image](https://github.com/user-attachments/assets/ea5a33a9-1995-4615-aa53-8d70bd5f7f02)

**Comment by flefevre** [comment]
Thanks for this hint
